src/components/select/SelectSelector.ts

```diff
@@ -15,7 +15,8 @@ export const getFilterText = (state: IReactVaporState, ownProps: ISelectWithFilt
     return (filter && filter.filterText) || '';
 };
 
-export const getListState = (state: IReactVaporState, ownProps: ISelectWithFilterProps): string[] => state.selectWithFilter[ownProps.id] ? state.selectWithFilter[ownProps.id].list : [];
+export const getListState = (state: IReactVaporState, ownProps: ISelectWithFilterProps): string[] =>

```

Comment:
IMO, I not agree with this kind of check because it's not a real use case to have an empty object for the React-Vapor State. In a real app, the redux store is alway initialized with all reducers and the * selectWithFilter* is always defined as an empty object (worst case).  If we need to add an check to make sure a UT passe. I think its the job of this UT to have the right State like the real app.